datafusion/proto/tests/cases/roundtrip_logical_plan.rs

```diff
@@ -375,6 +375,32 @@ async fn roundtrip_logical_plan_sort() -> Result<()> {
     Ok(())
 }
 
+#[tokio::test]
+async fn roundtrip_logical_plan_dml() -> Result<()> {
+    let ctx = SessionContext::new();
+
+    let schema = Schema::new(vec![
+        Field::new("a", DataType::Int64, true),
+        Field::new("b", DataType::Decimal128(15, 2), true),
+    ]);
+
+    ctx.register_csv(
+        "t1",
+        "tests/testdata/test.csv",
+        CsvReadOptions::default().schema(&schema),
+    )
+    .await?;
+
+    let query = "INSERT INTO T1 VALUES (1, null)";
```

Comment:
Can you please add at least basic coverage for the other DMLs? I realize there the actual code covered is minor (a different enum) but I think it will make sure we don't break it by accident and should be pretty straight forwrd

Here are some examples I think you could just follow:

https://github.com/apache/datafusion/blob/f3b1141d0f417e9d9e6c0ada03592c9d9ec60cd4/datafusion/core/tests/user_defined/insert_operation.rs#L42-L53